dubbo-samples-protosubff/dubbo-samples-protosubff-provider/src/main/java/org/dubbo/samples/protosubff/provider/service/UserService.java

```diff
@@ -0,0 +1,27 @@
+package org.dubbo.samples.protosubff.provider.service;
+
+import java.util.UUID;
+
+import org.apache.dubbo.config.annotation.Service;
+import org.dubbo.samples.protosubff.api.IUserService;
+import org.dubbo.samples.protosubff.domain.UserInfo;
+
+/**
```

Comment:
* use English to comment
* add apache license header
* remove author info

Reply:
Thank you very much for the fact that I have complied with the request and submitted it.